components/FilterCategories/FilterCategories.component.html

```diff
@@ -43,7 +43,12 @@ <h3 class="filter-categories__title">{{ $t('Filter') }}</h3>
           @click="onToggleCategory(category.external_id)"
           class="filter-categories__item_title"
           :class="{'filter-categories__item_title--hide': category.selected}"
-        >{{ category.title }}</h4>
+         v-if="category.title_translations == undefined">{{ category.title }}</h4>
```

Comment:
Brave attempt, but a few things wrong here:

1. Always use === for comparisons. That prevents weird things because "1" == 1 is true while "1" === 1 is not
2. Your code isn't really reflecting your intent (I think). You want to check if title_translations contains any values and your code will pass if title_translations = {} or title_translations = null, because both are "defined". There are alternatives. _.isNil checks if your var is undefined or defined as null. _.isEmpty checks if a container ({} included) has values set. You probably need both to check if there are translations.
3. If you really want to check for undefined use _.isUndefined or typeof foo === "undefined". Both are better than your approach, because if foo is truly undefined (as a variable) then it won't raise errors. I'll admit that when checking properties your code also doesn't throw errors, but it's nice to be consistent with these things.
4. The last point is not a blocker, but as you are rewriting let's make it a bit better. Your logic should exist in your JS not in your html. A category needs a hasTranslations method to tell the world (and your html templates) what its state is. Such methods may prove useful in the future, especially when the mechanic of translations changes in the future (somewhat hypothetically). Then you only need to rewrite the function and not a bunch of templates.

I hope this makes you value Python even more ;)